draft-ietf-quic-http.md

```diff
@@ -857,14 +909,17 @@ HTTP_INTERRUPTED_HEADERS (0x0E):
 : A HEADERS frame without the End Header Block flag was followed by a frame
   other than HEADERS.
 
-HTTP_SETTINGS_ON_WRONG_STREAM (0x0F):
-: A SETTINGS frame was received on a request control stream.
+HTTP_WRONG_STREAM (0x0F):
+: A frame was received on a different stream to where it is permitted.
```

Comment:
I learn something every day.  I say that in its most Lovecraftian meaning, of course.